content/posts/networkx/GSoC-2022/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "GSoC 2022"
+date: 2022-06-09
+draft: false
+description: "Getting started"
+tags: ["gsoc", "networkx"]
+displayInList: true
+author: ["kpetridis24"]
+
+resources:
+  - name: featuredImage
+    src: "GSoC_networkx_logo.png"
+    params:
+      description: "Google Summer of Code Logo with NetworkX logo"
+      showOnTop: true
+---
+
+# Intro
+I got accepted as a **GSoC** contributor, and I am so excited to spend the summer working on 
+such an incredibly interesting project. The mentors are very welcoming, communicative, fun 
+to be around, and I really look forward to collaborating with them. My application for GSoC 2022 
+can be [found here](https://summerofcode.withgoogle.com/programs/2022/projects/V1hY83XG).
+
+# About me
+My name is Konstantinos Petridis, and I am an **Electrical Engineering** student at the Aristotle University of 
+Thessaloniki. I am currently on my 5th year of studies, with a **Major in Electronics & Computer Science**. Although a 
+wide range of scientific fields fascinate me, I have a strong passion for **Computer Science**, **Physics** and **Space**.
+I love to study, learn new things and don't hesitate to express my curiosity by asking a bunch of questions to the point 
+of being annoying :).  
```

Comment:
A bit neutral, but this is not rendered. You can add a real smiley as we should support utf8 chars.